### PR TITLE
Logger message updated .

### PR DESCRIPTION
**Pull request by Novfal-202**
1.Logger message updated .
2.question set id in question and content id in question set updated as identifier.
3.model fields updated based on internal discussion.

### CODE REVIEW

File: src/controllers/bulkUpload.ts [hunk 1]
```diff
@@ -265,15 +266,15 @@ const handleCSVFiles = async (csvFiles: { entryName: string }[]) => {
     logger.info(`Content Validate::csv Data validation initiated for contents`);
     const contentsCsv = await handleContentCsv(validCSVData.contents, mediaEntries, processId);
     if (!contentsCsv.result.isValid) {
-      logger.error('content:: Error in question csv validation');
+      logger.error(contentsCsv.error.errMsg);
```
Comment (HarishGangula) [inline src/controllers/bulkUpload.ts:269]: use `optional chaining`  here to not get the error

File: src/controllers/bulkUpload.ts [hunk 2]
```diff
@@ -265,15 +266,15 @@ const handleCSVFiles = async (csvFiles: { entryName: string }[]) => {
...
       await ContentStage.destroy({ where: { process_id: processId } });
       return contentsCsv;
     }
 
     logger.info(`Question Set Validate::csv Data validation initiated for question sets`);
     const questionSetsCsv = await handleQuestionSetCsv(validCSVData.questionSets, processId);
-    if (!questionSetsCsv) {
-      logger.error('Question set:: Error in question set csv validation');
+    if (!questionSetsCsv.result.isValid) {
```
Comment (HarishGangula) [inline src/controllers/bulkUpload.ts:276]: use optional chaining here to not get the error

File: src/controllers/bulkUpload.ts [hunk 3]
```diff
@@ -265,15 +266,15 @@ const handleCSVFiles = async (csvFiles: { entryName: string }[]) => {
...
+      logger.error(questionSetsCsv.error.errMsg);
```
Comment (HarishGangula) [inline src/controllers/bulkUpload.ts:277]: use optional chaining here to not get the error

File: src/controllers/bulkUpload.ts [hunk 4]
```diff
@@ -283,7 +284,7 @@ const handleCSVFiles = async (csvFiles: { entryName: string }[]) => {
     logger.info(`Question Validate::csv Data validation initiated for questions`);
     const questionsCsv = await handleQuestionCsv(validCSVData.questions, mediaEntries, processId);
     if (!questionsCsv.result.isValid) {
-      logger.error('Question:: Error in question csv validation');
+      logger.error(questionsCsv.error.errMsg);
```
Comment (HarishGangula) [inline src/controllers/bulkUpload.ts:287]: use optional chaining here to not get the error

File: src/controllers/content.ts [hunk 5]
```diff
@@ -108,7 +97,7 @@ const validateCSVContentHeaderRow = async (contentEntry: any) => {
 
   const isValidHeader = validateHeader(contentEntry.entryName, header, templateHeader.result.data);
   if (!isValidHeader.result.isValid) {
-    logger.error('Content Row/Header:: Header validation failed');
+    logger.error(isValidHeader.error.errMsg);
```
Comment (HarishGangula) [inline src/controllers/content.ts:100]: use optional chaining here to not get the error

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: src/controllers/content.ts (hunk 5 above)
Comment (Novfal-202) [reply src/controllers/content.ts]: updated

File: src/controllers/bulkUpload.ts (hunk 4 above)
Comment (Novfal-202) [reply src/controllers/bulkUpload.ts]: updated

File: src/controllers/bulkUpload.ts (hunk 3 above)
Comment (Novfal-202) [reply src/controllers/bulkUpload.ts]: updated

File: src/controllers/bulkUpload.ts (hunk 2 above)
Comment (Novfal-202) [reply src/controllers/bulkUpload.ts]: updated

File: src/controllers/bulkUpload.ts (hunk 1 above)
Comment (Novfal-202) [reply src/controllers/bulkUpload.ts]: updated

**Review by HarishGangula — APPROVED** [review]

**Pull request merged**